### ISSUE TITLE
Feature request: fix scale for graphs

### ISSUE THREAD

**Issue by Alexander-Serov** [paradoxxxzero/gnome-shell-system-monitor-applet#402]
Thanks for the great extension! I think it could make even more useful if one could set a fixed scale for some of the indicators. Otherwise, for instance, by just looking at the network indicator filled up, it is hard to understand, whether my network is very fast or very slow. I just see that something is going on. 

I understand, this is not on your priority list, but still...

**Comment by franglais125** [comment]
I don't know that I'll have time for this (perhaps someone else can propose a PR), but at least you can use the tool tip for this. Just hovering the mouse on the graph should give you the speed on a pop-over.

**Comment by Alexander-Serov** [comment]
You are right, that's what I end up doing. But just something to keep in mind...

**Comment by TimP69** [comment]
I'd like to see this implemented as well.  Net is using/defaulting to a scale that is about 50% higher than the maximum speed of my ADSL.  As a result, the graph produced only occupies the bottom two thirds and it's hard (nearly impossible) to notice additional information (e.g. errors).

**Comment by rbaig** [comment]
I also would like to see a fixed scale implemented and came here exactly for this. Thanks for the extension.

**Comment by NickSto** [comment]
Yeah, I've increasingly realized that the network graph in particular isn't very useful without a fixed scale. I see something, and I often have no idea if it's a lot of traffic, or nearly zero traffic.

**Comment by hnb1** [comment]
Hi, I found this issue for a similar reason. with a CPU-graph perspective. The CPU graph would also  more helpful with a fixed scale: 
On my computer, the graph uses only half of the graph height and rescales in idle/heavy use cycles. Therefore, it's difficult to tell whether the CPU has been running at high or low capacity.
For the CPU graph, it suffices to have the graph maximum at maximum possible CPU freq.
No manual scale fixing would be needed if system monitor determines& sets max cpu speed on its own (maybe a ui flag to choose between autoscale/ CPU max scale would be nice).

**Comment by AllesMeins** [comment]
Since obviously nothing is happening to this feature-request, I dug into the code. Turns out a fixed scale is pretty easy to realize as an "ugly hack". But aware, it works for me but I don't know if it will break anything for you. So make a backup before changing any files! To get an fixed scale you just need to add one line of code to **extension.js** (located in _/home/<YOUE_USERNAME>/.local/share/gnome-shell/extensions/system-monitor@paradoxxx.zero.gmail.com/_)

Search for **const Disk = class SystemMonitor_Disk extends ElementBase {** in the file and add one line at the following place:

`const Disk = class SystemMonitor_Disk extends ElementBase {
    constructor() {
        super({
            elt: 'disk',
            item_name: _('Disk'),
            color_name: ['read', 'write']
        });
        
        this.max = 60; // ADD THIS LINE HERE. THIS WILL MAX OUT THE DISK-GRAPH AT 60 MB/s 
        
        this.mounts = MountsMonitor.get_mounts();`

Of cause you can change the value as you see fit for your system/needs.

You can do the same for the net-graph. Search for **const Net = class SystemMonitor_Net extends ElementBase {**

`const Net = class SystemMonitor_Net extends ElementBase {
    constructor() {
        super({
            elt: 'net',
            item_name: _('Net'),
            color_name: ['down', 'downerrors', 'up', 'uperrors', 'collisions']
        });
        
        this.max = 3000; // ADD THIS LINE HERE. THIS WILL MAX OUT THE NET-GRAPH AT 3 MB/s 
        
        this.speed_in_bits = false;`

**Comment by NickSto** [comment]
@AllesMeins This is incredible! Can't believe it was so easy. Thanks so much!

**Comment by core-genomix** [comment]
Yes! Right now my swap is at 50% but from the graph it seems it is at 100%!

![image](https://user-images.githubusercontent.com/96354770/235151070-608effd0-3076-4e51-8cf0-c4b24dfc8870.png)

**Comment by ZimbiX** [comment]
@core-genomix That sounds like a bug, as the swap graph does have a fixed scale of 100%. Do you have multiple swap files/partitions perhaps, and it doesn't show that correctly?
